infra/base-images/base-builder/jcc/jcc.go

```diff
@@ -341,6 +341,18 @@ func CppifyHeaderIncludes(contents string) (string, error) {
 	return contents, nil
 }
 
+func appendStringToFile(filepath, new_content string) error {
+    // Appends |new_content| to the content of |filepath|.
```

Comment:
formatting seems off. can you run go format.

Reply:
Done!